README.md

```diff
@@ -75,12 +74,11 @@ To learn more about Cortex, consult the following documents / talks:
 
 ## <a name="help"></a>Getting Help
 
-If you have any questions regarding Cortex, or on using Prometheus with Weave Cloud:
+If you have any questions regarding Cortex:
 
 
-- Ask a question on the <a href="https://weave-community.slack.com/messages/general/"> #weave-community</a> Slack channel. To invite yourself to the Weave Community Slack channel, visit https://weaveworks.github.io/community-slack/.
-- Join the <a href="https://www.meetup.com/pro/Weave/"> Weave User Group </a> and get invited to online talks, hands-on training and meetups in your area.
-- Send an email to <a href="mailto:weave-users@weave.works">weave-users@weave.works</a>
-- <a href="https://github.com/weaveworks/cortex/issues/new">File an issue.</a>
+- Ask a question on the [Cortex Slack channel](https://cloud-native.slack.com/messages/cortex/). To invite yourself to the CNCF Slack, visit http://slack.cncf.io/.
+- <a href="https://github.com/cortexproject/cortex/issues/new">File an issue.</a>
+Send an email to <a href="mailto:cortex-users@lists.cncf.io">cortex-users@lists.cncf.io</a>
```

Comment:
It's not listed here yet: https://lists.cncf.io/g/main/subgroups

Reply:
Chris said the request had been made.  Nobody uses email these days anyway ;-)

Reply:
:sadpanda: